src/ImageSharp/Formats/Tiff/TiffFrameMetadataExtensions.cs

```diff
@@ -31,7 +31,7 @@ public static T[] GetArray<T>(this TiffFrameMetadata meta, ExifTag tag, bool opt
         public static bool TryGetArray<T>(this TiffFrameMetadata meta, ExifTag tag, out T[] result)
```

Comment:
right, now can remove or simplify these methods

Reply:
@JimBobSquarePants  I got rid of some methods, but most remained, they are useful as wrappers for `ExifProfile` methods - allow you to cast types for properties

Reply:
>allow you to cast types for properties

I don't understand. The values returned by the existing method are already strong typed to the correct type without the need for internal calls and casting.

For example. This method is called upstream by the following.

```c#
byte[] buf = frame.GetArray<byte>(ExifTag.XMP, true);
if (buf != null)
{
    tiffMetadata.XmpProfile = buf;
}
```

However, this could easily have been.

```c#
IExifValue<byte[]> xmp = frame.ExifProfile.GetValue(ExifTag.XMP);
if (xmp != null)
{
    tiffMetadata.XmpProfile = xmp.Value;
}
```

which uses the existing methods removing any code duplication and is better for someone reading the code since it indicates exactly where the metadata is coming from.

I do wish that `ExifProfile.GetValue` was `ExifProfile.TryGetValue` but we can't fix that without introducing a breaking change (_I could be convinced to do that though_).

Reply:
The same duplication is happening in the resolution getter/setter code. 

We have existing methods in jpeg `GetExifResolutionValue` that demonstrate a much more simple approach using existing APIs.

Reply:
@JimBobSquarePants yes, you are right, it can be simplified even more, I did it

Reply:
Ace... I'll give it another review tonight. 👍